src/markdown-pages/install-with-kurl/system-requirements.md

```diff
@@ -47,55 +47,27 @@ For these OSes, the following packages are required per add-on:
 
 ## Disk Space Requirements
 
-### Core Requirements
+### Overall Disk Space Requirements
```

Comment:
Could go with something more specific for this heading like: "Per Node Disk Space" or "Node Disk Space" or "Minimum Node Disk Space"

Could also consider removing the word "Requirements" since it might be implied by being nested under the Disk Space Requirements H2. Alternatively, to leave in the word, recommend making it singular (Requirement) since there's only one requirement listed in this section

Reply:
Fixed, please take a look.